src/pages/LandingPage/LandingPage.styled.ts

```diff
@@ -1,3 +1,12 @@
-import styled from "styled-components";
+import styled from '@emotion/styled';
 
-export const LandingPageWrapper = styled.section``;
+export const LandingPageWrapper = styled.section`
```

Comment:
Couldnt you use styled component from MUI (@emotion/styled) rather than from styled-components